### ISSUE TITLE
Migrate to PPAPI rewrite

### ISSUE THREAD

**Issue by js0mmer** [icssc/peterportal-client#344]
Migrate to PPAPI rewrite once its ready.

Should resolve the following data inconsistency issues:
- #343 
- #238 
- #301 

and should unblock #224

**Comment by KevinWu098** [comment]
Whenever Peterportal Client wants to start migrating over to the new PPAPI rewrite, I'd be happy to handle the updates to time formatting! [I've already done the same over on AntAlmanac :)](https://github.com/icssc/AntAlmanac/pull/679)
